fix: Update to bootstrap 4.3.1 or higher

morganm58 opened the pull request:
fixes GH-9066

morganm58 committed:
feature/GH-9066
updated bootstrap in the package files.

morganm58 committed:
Merge branch 'develop' into feature/GH-9066

# Conflicts:
#	projects/storefrontlib/package.json

tobi-or-not-tobi reviewed (changes requested):
We need to do this differently: 
- change the root package json, use a caret in front of the dependency and bump to the minimum version we like to support. I'd go for `^4.5`
- update the lock file to ensure that the pipeline will use minimum 4.5, as we currently have the audit vulnerability. 
- change the public package.json files to support 4.5 as the minimum. Since this is part of our next 3.0 release, we can do the "breaking change".

morganm58 committed:
feature/GH-9066
updated bootstrap to use 4.5.2 as the minimum

morganm58 committed:
Merge branch 'develop' into feature/GH-9066

morganm58 committed:
feature/GH-9066
Updated yarn and package files to 4.5.3

morganm58 committed:
Merge branch 'develop' into feature/GH-9066

morganm58 committed:
feature/GH-9066
removed bootstrap dependency from storefrontlib and changed the version dependency on the public and storefrontstyles

morganm58 committed:
Merge branch 'develop' into feature/GH-9066

# Conflicts:
#	projects/storefrontlib/package.json

morganm58 committed:
Merge branch 'develop' into feature/GH-9066

morganm58 committed:
Merge branch 'develop' into feature/GH-9066

morganm58 committed:
Merge branch 'develop' into feature/GH-9066
